tests/e2e/config/bootstrap.js

```diff
@@ -167,11 +167,31 @@ function observeConsoleLogging() {
 			text.startsWith( 'Failed to decode downloaded font:' ) ||
 			text.startsWith( 'OTS parsing error:' ) ||
 			text.includes( 'Download the React DevTools for a better development experience' ) ||
-			text.includes( 'https://fb.me/react-unsafe-component-lifecycles' )
+			text.includes( 'Can\'t perform a React state update on an unmounted component' ) ||
```

Comment:
Minor nitpick, but 
```suggestion
			text.includes( "Can't perform a React state update on an unmounted component" ) ||
```
is a bit nicer and usually how we deal with quote marks in strings 😅